Classes/Controller/BackendServiceController.php

```diff
@@ -476,24 +478,28 @@ public function changeBaseWorkspaceAction(string $targetWorkspaceName, string $d
         // todo ensure that https://github.com/neos/neos-ui/pull/3734 doesnt need to be refixed in Neos 9.0
         $redirectNode = $documentNode;
         while (true) {
```

Comment:
this code is probably broken but i didnt touch it here ... this deserves a follwup